dash/frontend/src/app/modules/private/pages/cluster/kube-hunter/kube-hunter-report-details/kube-hunter-report-details.component.html

```diff
@@ -19,6 +19,7 @@
               </div>
             </mat-card-header>
             <mat-card-content>
+              <div class="page-table">
```

Comment:
Why is this needed?

Reply:
This class was created when I was trying different things to make the table to show correctly in a smaller screen. I forgot to clean up. Since there's a overflow in this class, I can make it a inline style.

Reply:
There are classes for overflowing cards.

`<mat-card class=".scrollable-table-card">`
`  <div class=table-wrapper>`
`    [mat-table element goes here]`
`  </div>`
`  <mat-paginator></mat-paginator>` (if applicable)
`</mat-card>`

Reply:
This isn't quite right yet. See my comment on where the classes should generally go.
It also doesn't scroll properly on a small screen yet.

Reply:
This is how the scroll looks like in a mobile screen.
![Screenshot 2023-07-24 at 2 16 54 PM](https://github.com/m9sweeper/m9sweeper/assets/56838011/5e716413-a2aa-4cb2-90b1-00d619cb9a66)